exp-player/addon/components/exp-player.js

```diff
@@ -4,12 +4,13 @@ import parseExperiment from '../utils/parse-experiment';
 
 export default Ember.Component.extend({
     layout: layout,
+    store: Ember.inject.service('store'),
```

Comment:
This was removed as part of the experiment refactor:
https://github.com/CenterForOpenScience/exp-addons/pull/19/files#diff-85b6be66a5ce62f39ee05c9d04ceb67fL6